imagedumper_desktop.go

```diff
@@ -12,40 +12,18 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//go:build !android && !ios && !js
-// +build !android,!ios,!js
+//go:build !android && !ios
```

Comment:
Now aren't imagedumper_desktop.go and imagedumper_none.go are almost the same?

Reply:
I'm confused, they're not the same - I moved dumpInternalImages out of _desktop (to _none and _js) to avoid the mkdir on js, if that's what you mean

Reply:
OK so please rename

* `imagedumper_desktop.go` -> `imagedumper_notmobile.go`
* `imagedumper_none.go` -> `imagedumper_desktop.go`

Reply:
gotcha.

Reply:
I think this file should be `imagedumper_notmobile.go`

Reply:
yeah, just realised that. fixing it now